### DISCUSSION TITLE
How can i add more Base Models if i'm using the online (google colab) version?

### DISCUSSION THREAD

**Discussion by MADELAT** [lllyasviel/Fooocus#2801] [Q&A] [answered]
Hello,

I've been using fooocus and in some videos i find on the net i see many people have got a variety of Base Models to chose from and try different results.

 I've just got jugernautSDL base model

How can i add more Base Models if i'm using the online (google colab) version?

Thank you

**Comment by Rogala** [comment, marked as answer]
[Setup and run Fooocus on Google Colab](https://youtu.be/07elsLQU6ho?si=2fvLDLAzB3zU_OWq)

**Reply by mashb1t** [reply]
@MADELAT you can use this video as reference or switch presets during runtime to automatically download their required models.
